prov/gni/src/gnix_cm_nic.c

```diff
@@ -464,7 +489,7 @@ int _gnix_cm_nic_enable(struct gnix_cm_nic *cm_nic)
 	if (cm_nic == NULL)
 		return -FI_EINVAL;
 
-	fabric = cm_nic->fabric;
+	fabric = cm_nic->domain->fabric;
```

Comment:
That's a fine idea but is not related to scalable endpoint so should be in a separate PR.